aperturedb/transformers/image_properties.py

```diff
@@ -0,0 +1,29 @@
+from aperturedb.transformers.transformer import Transformer
+from aperturedb.Subscriptable import Subscriptable
+from PIL import Image
+import traceback
+import io
+
+
+class ImageProperties(Transformer):
+    def __init__(self, data: Subscriptable, **kwargs) -> None:
+        super().__init__(data)
+
+    def getitem(self, subscript):
+        x = self.data[subscript]
+        try:
+            # x is a transaction that has an add_image command and a blob
+            for ic in self._add_image_index:
+                # Compute the dynamic properties and apply them to metadata
+                x[0][ic]["AddImage"]["properties"]["adb_image_size"] = len(
+                    x[1][ic])
```

Comment:
very cool, was thinking this would be nice to have. these could be useful in scripts as well as the commandline

kind of like a "middleware" .. 

```
qr = ParallelQuery
qr.ingest( csv, transformers = [ ImageProps, Resizer ] )
```

Reply:
This reads so easy on the eyes.
I am not yet doing it, as it needs to be tested on DaskManager.

Reply:
yeah, for sure later. I'll add a issue for feature

https://github.com/aperture-data/aperturedb-python/issues/326